dashboard/src/pages/TreeDetails/Tabs/Build/BuildTab.tsx

```diff
@@ -300,22 +319,34 @@ const BuildTab = ({ treeDetailsData }: BuildTab): JSX.Element => {
               }
               filters={[
                 {
-                  label: intl.formatMessage({ id: 'global.all' }),
+                  label: getFilterCountLabel(
+                    intl.formatMessage({ id: 'global.all' }),
+                    filterCount[possibleBuildsTableFilter[2]],
+                  ),
                   value: possibleBuildsTableFilter[2],
                   isSelected: selectedFilter === possibleBuildsTableFilter[2],
                 },
                 {
-                  label: intl.formatMessage({ id: 'global.successful' }),
+                  label: getFilterCountLabel(
+                    intl.formatMessage({ id: 'global.successful' }),
+                    filterCount[possibleBuildsTableFilter[1]],
+                  ),
                   value: possibleBuildsTableFilter[1],
                   isSelected: selectedFilter === possibleBuildsTableFilter[1],
                 },
                 {
-                  label: intl.formatMessage({ id: 'global.failed' }),
+                  label: getFilterCountLabel(
+                    intl.formatMessage({ id: 'global.failed' }),
+                    filterCount[possibleBuildsTableFilter[0]],
+                  ),
                   value: possibleBuildsTableFilter[0],
                   isSelected: selectedFilter === possibleBuildsTableFilter[0],
                 },
                 {
-                  label: intl.formatMessage({ id: 'global.inconclusive' }),
+                  label: getFilterCountLabel(
```

Comment:
you don't need a function for this, the format message has a template functionality in case another localization replaces the order https://formatjs.github.io/docs/getting-started/message-declaration

Reply:
nice catch!